Package path ./debug is not exported from package

When trying to import the debug module, I get the following error (removed `<path>` for clarity):

```
Module not found: Error: Package path ./debug is not exported from package <path>\node_modules\robot3 (see exports field in <path>\node_modules\robot3\package.json)
```

Looking at the `package.json` bundled into the package, it indeed does not have a mapping for it in `exports`:

```json
  "main": "dist/machine.js",
  "exports": {
    ".": {
      "require": "./dist/machine.js",
      "import": "./machine.js",
      "default": "./machine.js"
    }
  },
```
A new clause for debug is needed:

```json
  "main": "dist/machine.js",
  "exports": {
    ".": {
      "require": "./dist/machine.js",
      "import": "./machine.js",
      "default": "./machine.js"
    }
    "./debug": {
      "require": "./dist/debug.js",
      "import": "./debug.js",
      "default": "./debug.js"
    }
  },
```

--- a/.changeset/famous-pandas-grab.md
+++ b/.changeset/famous-pandas-grab.md
@@ -1,0 +1,5 @@
+---
+"robot3": patch
+---
+
+Add debug to package exports

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -7,10 +7,14 @@
   "exports": {
     ".": {
       "require": "./dist/machine.js",
- 			"types": "./index.d.ts",
       "import": "./machine.js",
       "default": "./machine.js",
       "types": "./index.d.ts"
+    },
+    "./debug": {
+      "require": "./dist/debug.js",
+      "import": "./debug.js",
+      "default": "./debug.js"
     }
   },
   "files": [

